javascript/javascript1/week4/voice_assistant.js

```diff
@@ -0,0 +1,94 @@
+const assistant = {};
+const addToDosList = (assistant.toDos = []);
+
+function getReply(command) {
```

Comment:
Consider passing the assistant as an argument for your getReply function -> getReply(command, assistant).
This would allow the function to be self-sufficient. You can check Example 2 from this link for a better explanation: https://github.com/HackYourFuture-CPH/curriculum/blob/main/review/review-examples.md